### PR TITLE
Update gradle to 7.2 and target & compile SDK to API 34

### PR DESCRIPTION
**Pull request by Seferi**
Hi there,

thanks for this package, it makes it very easy to capture audio. I updated my app to the API 34 and tried to build it with the latest flutter version 3.22.3 and your package gave some errors. Upgrading gradle and a few things a long fixed the issue. Here is a PR to make the app compatible with the latest flutter version and API 34. I'll be using the fork until it's merged. Thanks in advance.

### CODE REVIEW

File: android/gradlew
Comment (Clon1998) [inline android/gradlew:1]: No need to include a gradlew?

File: android/gradlew.bat
Comment (Clon1998) [inline android/gradlew.bat:1]: No need for the gradlew.bat

**Comment by Clon1998** [comment]
@ysak-y can you have a look at this. 

The plugin now prevents the build (Flutter v 3.24.x)

**Review by ysak-y — APPROVED** [review]
@Seferi @Clon1998 Sorry for the really late reply. It works with these commits if I attempt to build and run example application. LGTM.

**Comment by ysak-y** [comment]
Thanks so much @Seferi @Clon1998 !! I published this package with version 1.1.8 .
https://pub.dev/packages/flutter_audio_capture/versions/1.1.8

**Comment by Seferi** [comment]
@ysak-y Glad I could help and thank you for the great package.

**Pull request merged**